arch/arm/boot/dts/overlays/edt-ft5406-overlay.dts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Device Tree overlay for RaspberryPi 7" Touchscreen panel
+ *
+ */
+
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "brcm,bcm2835", "brcm,bcm2708", "brcm,bcm2709";
+
+	fragment@0 {
+		target = <&i2c_csi_dsi>;
+		__overlay__ {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "okay";
+			ft5406: ts@38 {
+				compatible = "edt,edt-ft5406";
+				reg = <0x38>;
+
+				touchscreen-size-x = < 800 >;
+				touchscreen-size-y = < 480 >;
+				touchscreen-inverted-x = < 0 >;
```

Comment:
Are these "inverted" flags booleans or integer properties? The defaults and the parameters disagree.

Reply:
Bools. That's present vs non-present then, isn't it.

Reply:
It is, annoyingly (if you're an overlay trying to set it to false and failing because you can't delete properties).

Reply:
Not an issue here as we just don't create it in this fragment, and let the override create it necessary, but yes, a general annoyance.